src/application/Fuzzer.ts

```diff
@@ -0,0 +1,970 @@
+
+      // Clone postman operation as new variation operation
+      const operationVariation = pmOperation.clone({
+        newId: camelCase(variationFuzzName),
+        name: variationFuzzName
+      })
+
+      // Set/Update Portman operation test type
+      this.testSuite.registerOperationTestType(
+        operationVariation,
+        PortmanTestTypes.variation,
+        false
+      )
+
+      // Change the value of the Postman the request property
+      const newVariation = JSON.parse(JSON.stringify(clonedVariation))
+      if (!newVariation?.overwrites) newVariation.overwrites = []
+
+      if (fuzzItems?.fuzzType === PortmanFuzzTypes.requestBody) {
+        const fuzzRequestBody = {
+          key: field.path,
+          value: numberVal,
+          overwrite: true
+        } as OverwriteRequestBodyConfig
+        this.addOverwriteRequestBody(newVariation, fuzzRequestBody)
+      }
+
+      if (fuzzItems?.fuzzType === PortmanFuzzTypes.requestQueryParam) {
+        const fuzzRequestQueryParam = {
+          key: field.path,
+          value: numberVal.toString(), // Query params should passed as string to Postman
+          overwrite: true
+        } as unknown as OverwriteQueryParamConfig
+        this.addOverwriteRequestQueryParam(newVariation, fuzzRequestQueryParam)
+      }
+
+      if (fuzzItems?.fuzzType === PortmanFuzzTypes.requestHeader) {
+        const fuzzRequestHeader = {
+          key: field.path,
+          value: numberVal.toString(),
+          overwrite: true
+        } as OverwriteRequestHeadersConfig
+        this.addOverwriteRequestHeader(newVariation, fuzzRequestHeader)
+      }
+
+      this.variationWriter.injectVariations(
+        operationVariation,
+        oaOperation,
+        newVariation,
+        variationMeta
+      )
+
+      // Build up list of Fuzz Variations
+      this.fuzzVariations.push(operationVariation)
+    })
+  }
+
+  public injectFuzzMaximumVariation(
+    pmOperation: PostmanMappedOperation,
+    oaOperation: OasMappedOperation | null,
+    variation: VariationConfig,
+    variationMeta: VariationTestConfig | IntegrationTest | null,
+    fuzzItems: FuzzingSchemaItems | null
+  ): void {
+    // Early exit if no fuzzing fields defined
+    const maximumNumberFields = fuzzItems?.maximumNumberFields || []
+    if (maximumNumberFields.length === 0) return
+    if (
+      !(PortmanFuzzTypes.requestBody === fuzzItems?.fuzzType) &&
+      !(PortmanFuzzTypes.requestQueryParam === fuzzItems?.fuzzType) &&
+      !(PortmanFuzzTypes.requestHeader === fuzzItems?.fuzzType)
+    )
+      return
+
+    const clonedVariation = JSON.parse(JSON.stringify(variation))
+
+    maximumNumberFields.forEach(field => {
+      // Set Pm request name
+      const variationFuzzName = `${pmOperation.item.name}[${variation.name}][maximum number value ${field.field}]`
+
+      // Transform to number
+      const numberVal =
+        typeof field.value === 'number' ? field.value + 1 : parseInt(field.value) + 1
```

Comment:
should be able to handle decimals too no? 

Ie: as soon as you call parseInt on `"10.01"` you'll be changing to `10`.